editor/scene_tree_editor.cpp

```diff
@@ -276,18 +276,27 @@ bool SceneTreeEditor::_add_nodes(Node *p_node, TreeItem *p_parent, bool p_scroll
 
 		String msg_temp;
 		if (num_connections >= 1) {
-			Array arr;
-			arr.push_back(num_connections);
-			msg_temp += TTRN("Node has one connection.", "Node has {num} connections.", num_connections).format(arr, "{num}");
-			msg_temp += " ";
+			msg_temp += vformat(TTRN("Node has one connection.", "Node has %d connections.", num_connections), num_connections);
```

Comment:
I think the previous `format()` call might have been because `vformat` would fail when `num_connections` is 1 and the translated string therefore has no `%d`. Does it work fine now?